src/util/ecdsa.rs

```diff
@@ -151,6 +168,26 @@ pub struct PrivateKey {
 }
 
 impl PrivateKey {
+    /// Constructs compressed ECDSA PrivateKey from the provided generic Secp256k1 private key
+    /// and the specified network
+    pub fn new(pk: secp256k1::SecretKey, network: Network) -> PrivateKey {
+        PrivateKey {
+            compressed: true,
+            network: network,
```

Comment:
Why not just `network,`?

Reply:
I don't know why but I prefer `network: network` instead of `network`, looks aesthetically cleaner to me when you have other fields that use the `a: b` syntax.

Reply:
Ok, I probably just got conditioned by my linter :smile:

Reply:
I was always simplifying `A: A` to `A`, but with the ancient rustc 1.29 I am not sure whether it knows about this stuff – and I am tired of CI failing just because you use rust syntactic sugar.